pkg/lint/rules/template.go

```diff
@@ -51,7 +54,17 @@ func Templates(linter *support.Linter) {
 	}
 
 	options := chartutil.ReleaseOptions{Name: "testRelease", Time: timeconv.Now(), Namespace: "testNamespace"}
-	caps := &chartutil.Capabilities{APIVersions: chartutil.DefaultVersionSet}
+	caps := &chartutil.Capabilities{
+		APIVersions: chartutil.DefaultVersionSet,
+		KubeVersion: &version.Info{
```

Comment:
I thought we'd want to have something resembling real data in that field for chart linting.